charts/datadog/README.md

```diff
@@ -412,7 +412,7 @@ helm install --name <RELEASE_NAME> \
 | clusterAgent.strategy | object | `{"rollingUpdate":{"maxSurge":1,"maxUnavailable":0},"type":"RollingUpdate"}` | Allow the Cluster Agent deployment to perform a rolling update on helm update |
 | clusterAgent.token | string | `""` | Cluster Agent token is a preshared key between node agents and cluster agent (autogenerated if empty, needs to be at least 32 characters a-zA-z) |
 | clusterAgent.tokenExistingSecret | string | `""` | Existing secret name to use for Cluster Agent token |
-| clusterAgent.useHostNetwork | string | `nil` | Bind ports on the hostNetwork |
+| clusterAgent.useHostNetwork | bool | `false` | Bind ports on the hostNetwork |
```

Comment:
is it backward  compatible? for example if some already set in their values.yaml `clusterAgent.useHostNetwork: "true"`
we should increase the 2 digits of the helm chart version

Reply:
The change here seems to be due to the documentation generation. I guess it defaults to `string` type when no value is given.

The field `useHostNetwork` has always been a `boolean`, so no changes here.